pkgs/development/python-modules/sphinxawesome-theme/default.nix

```diff
@@ -0,0 +1,41 @@
+{
+  buildPythonPackage,
+  fetchPypi,
+  lib,
+  poetry-core,
+  sphinx,
+  beautifulsoup4,
+  pythonRelaxDepsHook
+}:
+
+buildPythonPackage rec {
+  pname = "sphinxawesome-theme";
+  version = "5.1.4";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit version;
+    pname = "sphinxawesome_theme";
+    hash = "sha256-OwikuKJrPo4vNaud/9JToYYJePV6Kew8izYbr/qKTtQ=";
+  };
+
+  nativeBuildInputs = [ pythonRelaxDepsHook ];
```

Comment:
I think you can safely merge `nativeBuildInputs` and `build-system`. Hope @mweinelt is OK with that :)